docs/index.md

```diff
@@ -1,23 +1,23 @@
 ---
-content_title: EOSIO Overview
+content_title: Antelope Overview
 ---
 
-EOSIO is the next-generation blockchain platform for creating and deploying smart contracts and distributed applications. EOSIO comes with a number of programs. The primary ones included in EOSIO are the following:
+Antelope is the next-generation blockchain platform for creating and deploying smart contracts and distributed applications. Antelope comes with a number of programs. The primary ones included in Antelope are the following:
 
 * [Nodeos](01_nodeos/index.md) - Core service daemon that runs a node for block production, API endpoints, or local development.
 * [Cleos](02_cleos/index.md) - Command line interface to interact with the blockchain (via `nodeos`) and manage wallets (via `keosd`).
-* [Keosd](03_keosd/index.md) - Component that manages EOSIO keys in wallets and provides a secure enclave for digital signing.
+* [Keosd](03_keosd/index.md) - Component that manages Antelope keys in wallets and provides a secure enclave for digital signing.
 
 The basic relationship between these components is illustrated in the diagram below.
 
-![EOSIO components](mandel_components.png)
+![Antelope components](mandel_components.png)
 
-Additional EOSIO Resources:
-* [EOSIO Utilities](10_utilities/index.md) - Utilities that complement the EOSIO software.  
+Additional Antelope Resources:
+* [Antelope Utilities](10_utilities/index.md) - Utilities that complement the Antelope software.  
 
 [//]: # (THIS IS A COMMENT REMOVING BROKEN LINKS)  
-[//]: # (Upgrade-Guide-20_upgrade-guide/index.md-EOSIO-version/protocol-upgrade-guide.)  
-[//]: # (Release Notes 30_release-notes/index.md  - All release notes for this EOSIO version.)  
+[//]: # (Upgrade-Guide-20_upgrade-guide/index.md-antelope-version/protocol-upgrade-guide.)  
```

Comment:
FYI these are commented out broken links

Reply:
Ok thanks for letting me know. I saw this format and I thought it was optional. I will fix it and keep it in mind in the future.